src/components/TodaySection.astro

```diff
@@ -132,7 +131,6 @@ const subtitles = [
 									clearInterval(bleachIntervalId)
 
 									envisionSection?.classList.remove("hidden");
-									todaySection?.classList.add('translate-y-0'); // Hack for remove border shadow between today and envision
```

Comment:
@GusevP  Let's understand what the hack I deleted. May you describe how can I look at it please